src/client/App.js

```diff
@@ -11,13 +11,32 @@ import Header from './components/NavigationHeader/NavigationHeader';
 import Profile from './containers/Profile';
 import Channel from './containers/Channel/Channel';
 import Loader from './components/Loader/Loader';
+import fetchWithAuth from './utils/fetchWithAuth';
 import { UserContext } from './context/userContext';
 
 function App() {
+  const [currentUser, setCurrentUser] = useState();
   const { isAuthenticated, isLoading, user } = useAuthentication();
+
+  useEffect(() => {
+    const fetchUserFromDatabase = async () => {
+      try {
+        const { uid } = await user;
+        const url = `/api/users/current/?uid=`;
+        const userFromDB = await fetchWithAuth(url + uid);
+        return setCurrentUser(userFromDB);
```

Comment:
You probably want to combine the data. Use spreading e.g.:
```
setCurrentUser({
  ...currentUser,
  ...userFromDB
});```

Reply:
In that case userFromDB I want just replace it because userFromDB has everything.